manifests/master.pp

```diff
@@ -389,11 +389,26 @@
       notify  => Service[$service]
     }
 
+    # `[master] ca` is deprecated, as of Puppet 5.5.6 (SIMP-5456), and removed
+    # in Puppet 6 (PUP-9158).  All
+    if versioncmp($facts['puppetversion'], '6.0') >= 0 {
+      $_ensure_master_ca = 'absent'
+    }
+    elsif versioncmp($facts['puppetversion'], '5.5.6') {
+      $_ensure_master_ca = $enable_ca ? {
+        true    => 'absent',
```

Comment:
In Puppet 5 in which there is no trapperkeeper, what is the implication of not having a master ca  setting (i.e., the ensure absent)?